corehq/apps/translations/app_translations/upload_app.py

```diff
@@ -239,18 +246,49 @@ def _process_single_sheet(app, sheet, names_map, lang=None):
         if not row['case_property'] and not row['list_or_detail'] and not row['label']:
             modules_and_forms_rows.append(row)
         elif module_or_form != row['menu_or_form']:
-            msgs.extend(_process_rows(app, module_or_form, rows, names_map, lang=lang))
+            msgs.extend(_process_rows(app, module_or_form, rows, names_map, lang=lang, is_single_sheet=True))
             module_or_form = row['menu_or_form']
             rows = [row]
         else:
             rows.append(row)
-    msgs.extend(_process_rows(app, module_or_form, rows, names_map, lang=lang))
+    msgs.extend(_process_rows(app, module_or_form, rows, names_map, lang=lang, is_single_sheet=True))
     msgs.extend(_process_rows(app, MODULES_AND_FORMS_SHEET_NAME,
-                              modules_and_forms_rows, names_map, lang=lang))
+                              modules_and_forms_rows, names_map, lang=lang, is_single_sheet=True))
     return msgs
 
 
-def _process_rows(app, sheet_name, rows, names_map, lang=None):
+def get_older_rows(app, lang, sheet_name, is_single_sheet):
```

Comment:
nit: should this method be defined after `_process_rows`? Breaks the order of method calls.